src/api/database/misc.rst

```diff
@@ -78,47 +63,216 @@
         {
             "c6114c65e295552ab1019e2b046b10e": [
                 "3-b06fcd1c1c9e0ec7c480ee8aa467bf3b",
-                "3-0e871ef78849b0c206091f1a7af6ec41"
+                "3-c50a32451890a3f1c3e423334cc92745"
             ]
         }
 
     **Response**:
 
     .. code-block:: http
 
-        HTTP/1.1 200 OK
+        HTTP/1.1 201 Created
         Cache-Control: must-revalidate
-        Content-Length: 103
+        Content-Length: 107
         Content-Type: application/json
-        Date: Mon, 12 Aug 2013 10:53:24 GMT
-        Server: CouchDB (Erlang/OTP)
+        Date: Fri, 02 Jun 2017 18:55:54 GMT
+        Server: CouchDB/2.0.0-2ccd4bf (Erlang OTP/18)
 
         {
-            "purge_seq":3,
```

Comment:
Did you intend to eliminate "purge_seq" from this example? The docs still mention it above (but now as a string, not a number)

Reply:
added again :-)